src/index.js

```diff
@@ -0,0 +1,91 @@
+let new_todo = document.getElementById('new-todo-title');
+let answer;
+
+new_todo.addEventListener("keyup", function(event) {
+    // Enter키를 누를시 실행
+    if (event.keyCode === 13) {
```

Comment:
```suggestion
    if (event.key === 'Enter') {
```
이 부분은 ```keyCode```를 사용하지 않고 ```key``` 요소를 사용한다면 주석없이 더 깔끔한 코드가 될거 같습니다!